products/apache-hop.md

```diff
@@ -16,9 +16,16 @@ auto:
 
 # eol(x) = releaseDate(x+1)
 releases:
+-   releaseCycle: "2.9"
+    releaseDate: 2024-05-20
+    eol: false
+    latest: '2.9.0'
+    latestReleaseDate: 2024-05-20
+    link: https://github.com/apache/hop/releases/tag/2.9.0-rc1
```

Comment:
gonna raise an issue about this

Reply:
for 2.8, the blog post was published on March 13, so yeah, it might take two weeks?